packages/@uppy/companion/src/server/provider/facebook/index.js

```diff
@@ -28,6 +19,47 @@ class Facebook extends Provider {
     return 'facebook'
   }
 
+  async runRequestBatch({ token, requests }) {
```

Comment:
I'd put `runRequestBatch` and `getMediaUrl` back outside of the class (by adding `secret` to `runRequestBatch` args), to make it clear that they don't use any covert class variables other than what's in the args.